Tests/SentryTests/Helper/TestNSNotificationCenterWrapper.swift

```diff
@@ -1,19 +1,27 @@
 import Foundation
 
-class TestNSNotificationCenterWrapper: SentryNSNotificationCenterWrapper {
+@objc public class TestNSNotificationCenterWrapper: SentryNSNotificationCenterWrapper {
```

Comment:
What was the reason for these `@objc public` additions?

Reply:
So I can pass `TestNSNotificationCenterWrapper` to an ObjC class.